app/workers/import_worker.rb

```diff
@@ -71,4 +73,18 @@ def process_follows
       end
     end
   end
+
+  def process_favourites
+    import_rows.each do |row|
+      begin
+        domain, account, id = row
+
+        status = (domain == Rails.configuration.x.local_domain) ? Status.find(id) : FetchRemoteStatusService.new.call("http://#{domain}/users/#{account}/#{id}.atom")
```

Comment:
Oh, ok. So is importing impractical at the moment? Maybe I should revert to c7bf4794d7149df3d29f4cad98cc2d4c42fb346b then.